Source/ablastr/math/fft/WrapRocFFT.cpp

```diff
@@ -66,6 +66,69 @@ namespace ablastr::math::anyfft
         return fft_plan;
     }
 
+    FFTplan CreatePlanMany(int * real_size, amrex::Real * real_array,
+                           Complex * complex_array, const direction dir, const int dim,
+                           int howmany, int * inembed, int istride, int idist,
+                           int * onembed, int ostride, int odist)
+    {
+        FFTplan fft_plan;
+
+        const std::size_t lengths[] = {AMREX_D_DECL(std::size_t(real_size[0]),
+                                                    std::size_t(real_size[1]),
+                                                    std::size_t(real_size[2]))};
```

Comment:
In this particular application, `real_size[2]` is not defined.
Maybe instead of using `AMREX_D_DECL`, we should do a loop over `dim-1` elements.